### ISSUE TITLE
Weekends and Holidays add on not rescheduling cards

### ISSUE THREAD

**Issue by Mckaycs** [AnKing-VIP/ankiweb-placeholder#77]
## Which add-on is this about?
Weekends and holidays

#### Problem description

*on version 23.10 for the last month with FSRS in use for the same amount of time. The add on was working great for that time, just rescheduling cards so I had Sundays off. I opened the add on to add a couple specific days around Jan 6 and 7 as holidays which worked appropriately. Now, whenever I open the add on, it's blank with no fields or check boxes filled out. When I select sunday (or any other day instead) and then select reschedule, it says "no cards rescheduled" and even though I select save, when I open the add on window again, each field and check box is blank again. None of my cards are being rescheduled now.

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [ x ] I've restarted Anki to see if it helps
 - [ x ] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [ x ] I've tried to disable other add-ons to see if there are any interactions present
 - [ Not true ] My issue disappears when I hold shift while starting Anki.
 - [ x ] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
23.10 with FSRS scheduling
```

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```
none
```

**Comment by AnKingMed** [comment]
So it's not saving any checked fields? You didn't update your Anki version correct? Did you install any other addons? Does it work if it's the only add-on installed?

Might be worth deleting it and redownloading to see if that fixes things

**Comment by Mckaycs** [comment]
Correct! And yes, I’m on version 23.10. I haven’t installed any other add ons in the last 4 or so months.

I just deleted and redownloaded it like you mentioned. When I ran it, it still said “0 cards rescheduled”, but when I checked my stats, it does show everything rescheduled so that I don’t have any cards on sunday.

So, it appears fixed on my end! I’m not sure what caused the issue in the first place, but thank you for your help with that!

Best,


Mckay Smith
OMS II 
***@***.***   |   (385) 204-1637
